pkg/backend/httpstate/client/client.go

```diff
@@ -1411,15 +1436,28 @@ func (pc *Client) SummarizeErrorWithCopilot(
 	// context deadline exceeded" Copilot backend will see this in telemetry as well
 	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
 	defer cancel()
+
 	url := pc.apiURL + "/api/ai/chat/preview"
+	apiToken := string(pc.apiToken)
+
+	// TODO: Debug helpers. Remove before merging.
+	copilotURL := os.Getenv("DEBUG_PULUMI_COPILOT_URL")
+	if copilotURL != "" {
+		url = copilotURL + "/api/ai/chat/preview"
+		apiToken = os.Getenv("DEBUG_PULUMI_COPILOT_TOKEN")
+	}
```

Comment:
Should these be removed now?

Reply:
Yes! I feel like we're getting close! 🤞 

Speaking of is there a existing way to do this kind thing? I will look into a better long term solution here. It might be a local pulumi-service instance.. 🤔 .

Reply:
Yeah, I think logging into a local pulumi-service instance would work.  But not sure tbh, I never worked on anything that required service updates, so I just use the regular pulumi instance.